float64 realFx
float64 realFy
float64 realFz
float64 realMx
float64 realMy
float64 realMz